test/e2e/mutate/config.go

```diff
@@ -123,13 +123,5 @@ var ingressTests = struct {
 			resourceName: "kuard-v1beta1",
 			resource:     ingressNetworkingV1beta1,
 		},
-		{
-			testName:     "test-extensions-v1beta1-ingress",
-			group:        "extensions",
-			version:      "v1beta1",
-			rsc:          "ingresses",
-			resourceName: "kuard-extensions",
-			resource:     ingressExtensionV1beta1,
-		},
```

Comment:
Is this test removed by accident?

Reply:
No. This version was deprecated in 1.16. We no longer need to support it. It fails when using the using 1.20+